docker/docker.go

```diff
@@ -161,6 +161,12 @@ func (d *ImageManager) FindImage(imageName string) (*dockerclient.Image, error)
 	return image, nil
 }
 
+// HasImage determines if the given image already exists in Docker
+func (d *ImageManager) HasImage(imageName string) bool {
+	_, err := d.FindImage(imageName)
+	return (err == nil)
```

Comment:
This is not 100% correct - see `FindImage` above.
Docker may be replying with other errors.
This helper is not proper. The callers will be ignoring errors.
We should remove this.